vm_network_migration/tests/test_roll_back.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test roll_back_original_instance() function
+"""
+
+import json
+import os
+from unittest import mock
+from unittest.mock import patch
+
+import google.auth.credentials
+import httplib2
+import unittest2 as unittest
+from googleapiclient.errors import HttpError
+from vm_network_migration.vm_network_migration import *
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
+
+
+def datafile(filename):
+    """Generate path of the file
+    Args:
+        filename: file name
+
+    Returns: the file path
+
+    """
+    return os.path.join(DATA_DIR, filename)
+
+
+def read_json_file(filename):
+    """Read *.json file
+    Args:
+        filename: json file name
+
+    Returns: a Python object
+
+    """
+    with open(datafile(filename)) as f:
+        res = json.load(f)
+        f.close()
+    return res
+
+
+@patch("vm_network_migration.vm_network_migration.attach_disk")  # index: 3
+@patch(
+    "vm_network_migration.vm_network_migration.wait_for_operation")  # index: 2
+@patch("vm_network_migration.vm_network_migration.start_instance")  # index: 1
+@patch("google.auth.default")  # index 0
+class RollBackOriginalInstance(unittest.TestCase):
```

Comment:
Add one more verification that the old instance is restored to what it was and that the new instance no longer exists.